### PR TITLE
Updated List API to retrieve ACTIVE findings only and increased pageSize

### LINKED ISSUE #3425: [v2.23.0] Google SCC notifier doesn't work, 500 error
Terraform module v5.0.0 + Forseti v2.23.0 on GCE.

My notifier run fails every time ~4 hours after start according to the log:

```python
2019-10-27 18:50:47,859 INFO [forseti-security][2.23.0] google.cloud.forseti.notifier.notifiers.cscc_notifier(run): Running Cloud Security Command Center notification module.
2019-10-27 22:54:53,175 ERROR [forseti-security][2.23.0] google.cloud.forseti.services.notifier.service(_run_notifier): <HttpError 500 when requesting https://securitycenter.googleapis.com/v1/organization
s/123456789000/sources/1234567890001112223/findings?alt=json&pageToken=CusC3cy7qgAx8QqflqlfsMwLpOnD9w5itomdonX0K1oIF9Brwj8Uztlw9lYqWvw9airRnWuQUWwnucxru6Mm1ydP3pxRVGEUuePu6mj4WTsZuhYEYXO94-KsIMJdvCOyGy-QG
hY9b1EZHoFwdD-_8gAwFcoWCLafn0DCBBg8L8ICJDhItQuBASTNtQEL8J3TUi0ocpjKmt3gwsI9f0aygXMTxsT5OSmdca0-OJaoFcOPLVaXZbXjMjugvBSWlxxLVb3q-X3ZbzXe3-AhfWTRht3LLtF8d644ywr5BTKQ_8qhAIIyjKCtaSDStoimQEGw5LFd6cW4ZSF6pgIM9
CbYJGjt-vkNhMmmuOwP8JuZDFRxestHXhFw0ZDZ0xmVF6hu4i3quvoFjlUZ4_EHI8vgaD2gE_behgFt-kgbj1daMIlE5hoZaaUQujRxKBULa4NgIXTQwxWU89aV8I-E38SyZMvwm5B1QUWwvvtDON5J returned "Internal error encountered.">
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/services/notifier/service.py", line 129, in _run_notifier
    self.service_config)
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/notifier/notifier.py", line 187, in run
    .run(violations_as_dict, source_id=source_id))
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/notifier/notifiers/cscc_notifier.py", line 285, in run
    self._send_findings_to_cscc(violations, source_id=source_id)
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/notifier/notifiers/cscc_notifier.py", line 229, in _send_findings_to_cscc
    for page in paged_findings_in_cscc:
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/common/gcp_api/repository_mixins.py", line 55, in list
    verb_arguments=arguments):
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/common/gcp_api/_base_repository.py", line 467, in execute_paged_query
    response = self._execute(request)
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/common/util/replay.py", line 169, in replay_wrapper
    return f(self, request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/common/util/replay.py", line 82, in record_wrapper
    return f(self, request, *args, **kwargs)
  File "/home/ubuntu/forseti-security/.eggs/retrying-1.3.3-py3.6.egg/retrying.py", line 49, in wrapped_f
    return Retrying(*dargs, **dkw).call(f, *args, **kw)
  File "/home/ubuntu/forseti-security/.eggs/retrying-1.3.3-py3.6.egg/retrying.py", line 206, in call
    return attempt.get(self._wrap_exception)
  File "/home/ubuntu/forseti-security/.eggs/retrying-1.3.3-py3.6.egg/retrying.py", line 247, in get
    six.reraise(self.value[0], self.value[1], self.value[2])
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 693, in reraise
    raise value
  File "/home/ubuntu/forseti-security/.eggs/retrying-1.3.3-py3.6.egg/retrying.py", line 200, in call
    attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
  File "/usr/local/lib/python3.6/dist-packages/forseti_security-2.23.0-py3.6.egg/google/cloud/forseti/common/gcp_api/_base_repository.py", line 541, in _execute
    num_retries=self._num_retries)
  File "/home/ubuntu/forseti-security/.eggs/google_api_python_client-1.7.10-py3.6.egg/googleapiclient/_helpers.py", line 130, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/home/ubuntu/forseti-security/.eggs/google_api_python_client-1.7.10-py3.6.egg/googleapiclient/http.py", line 855, in execute
    raise HttpError(resp, content, uri=self.uri)
googleapiclient.errors.HttpError: <HttpError 500 when requesting https://securitycenter.googleapis.com/v1/organizations/123456789000/sources/1234567890001112223/findings?alt=json&pageToken=CusC3cy7qgAx8Qq
flqlfsMwLpOnD9w5itomdonX0K1oIF9Brwj8Uztlw9lYqWvw9airRnWuQUWwnucxru6Mm1ydP3pxRVGEUuePu6mj4WTsZuhYEYXO94-KsIMJdvCOyGy-QGhY9b1EZHoFwdD-_8gAwFcoWCLafn0DCBBg8L8ICJDhItQuBASTNtQEL8J3TUi0ocpjKmt3gwsI9f0aygXMTxsT
5OSmdca0-OJaoFcOPLVaXZbXjMjugvBSWlxxLVb3q-X3ZbzXe3-AhfWTRht3LLtF8d644ywr5BTKQ_8qhAIIyjKCtaSDStoimQEGw5LFd6cW4ZSF6pgIM9CbYJGjt-vkNhMmmuOwP8JuZDFRxestHXhFw0ZDZ0xmVF6hu4i3quvoFjlUZ4_EHI8vgaD2gE_behgFt-kgbj1d
aMIlE5hoZaaUQujRxKBULa4NgIXTQwxWU89aV8I-E38SyZMvwm5B1QUWwvvtDON5J returned "Internal error encountered.">
```

As a result I never see any event updates in Google SCC.

As per @hshin-g suggestion, I've decreased the number of calls for `securitycenter_max_calls` to `10`  in the `notifier` section (https://github.com/forseti-security/terraform-google-forseti/blob/e3ed110e80ee42b87f6ede1fa348c139d479dd1f/modules/server_config/templates/configs/forseti_conf_server.yaml.tpl#L277) and re-run the notifier, I'll report if that will help.

### PR DESCRIPTION
**Pull request by red2k18**
Resolves #3425, #2661 

This PR retrieve active findings only from CSCC and increases pageSize.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [x] All of the [tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) pass

### CODE REVIEW

File: google/cloud/forseti/common/gcp_api/securitycenter.py [hunk 1]
```diff
@@ -155,6 +155,7 @@ def create_finding(self, finding, source_id=None, finding_id=None):
                     finding.get('source_properties').get('violation_data'))
                 raise api_errors.ApiExecutionError(violation_data, e)
 
+    # pylint: disable=logging-too-many-args
```
Comment (joecheuk) [inline google/cloud/forseti/common/gcp_api/securitycenter.py]: why is this pylint disable needed?

File: google/cloud/forseti/common/gcp_api/securitycenter.py [hunk 2]
```diff
@@ -165,8 +166,18 @@ def list_findings(self, source_id):
...
         """
-        response = self.repository.findings.list(parent=source_id)
-        return response
+
+        try:
+            LOGGER.debug('Listing findings.')
+
+            response = self.repository.findings.list(parent=source_id,
+                                                     filter='state="ACTIVE"',
+                                                     pageSize=100)
```
Comment (joecheuk) [inline google/cloud/forseti/common/gcp_api/securitycenter.py]: Any reason why the page size is 100?

Comment (red2k18) [reply google/cloud/forseti/common/gcp_api/securitycenter.py]: Page size can be anywhere between 1 and 1000. It's the default (10) right now. I have bumped it to 100 as it seems to be higher than we we had (10) but not too high.

Comment (joecheuk) [reply google/cloud/forseti/common/gcp_api/securitycenter.py]: Any reasons to not maxing it out at 1000? is there any disadvantage to do so compare to say 100?

File: google/cloud/forseti/common/gcp_api/securitycenter.py (hunk 1 above)
Comment (red2k18) [reply google/cloud/forseti/common/gcp_api/securitycenter.py]: Removed pylint disable by editing logging statement.

File: google/cloud/forseti/common/gcp_api/securitycenter.py (hunk 2 above)
Comment (red2k18) [reply google/cloud/forseti/common/gcp_api/securitycenter.py]: It can be anywhere between 10 and 1000 based on need/preference.

Comment (joecheuk) [reply google/cloud/forseti/common/gcp_api/securitycenter.py]: I would change this to 1000 if there isnt any disadvantage of doing so.

Comment (red2k18) [reply google/cloud/forseti/common/gcp_api/securitycenter.py]: Updated to 1000.

**Comment by red2k18** [comment]
Cleaned it up before opening the PR. All the changes are related. Let me know if you have any question on any change.

**Comment by joecheuk** [comment]
@gkowalski-google what do you think about merging this to master?

**Comment by gkowalski-google** [comment]
> @gkowalski-google what do you think about merging this to master?

@joecheuk @red2k18 It's fine to merge to master, but it means that it won't make it into the v2.25.0 release. I think the reason why @joecheuk saw so many changes is because the branch was created from master, which has a lot of changes in it that are not in the release. If the intent was to include this change in the release, then it would be easier to create a new branch from the current release branch and bring over only the relevant changes needed for this change.

**Comment by red2k18** [comment]
@gkowalski-google an alternative would be to cherry-pick the squashed commit on the release branch. What are your thoughts on that?

**Comment by gkowalski-google** [comment]
> @gkowalski-google Alternatively, the squashed commit can be cherry-picked on the release branch. What are your thoughts on that?

Either way is fine with me.

**Pull request merged**